### PR TITLE
Fixed tooltip label and date format of milestone line for new date types

### PR DESCRIPTION
**Pull request by savvinsergey**

### CODE REVIEW

File: src/gantt.ts [hunk 1]
```diff
@@ -1407,10 +1407,27 @@ module powerbi.extensibility.visual {
...
 
-        private getTooltipForMilstoneLine(timestamp: number, milestoneTitle: string): VisualTooltipDataItem[] {
+        private getTooltipForMilstoneLine(
+            timestamp: number,
+            milestoneTitle: string): VisualTooltipDataItem[] {
+            let dateTime: string = new Date(timestamp).toLocaleDateString();
+
+            if (!milestoneTitle) {
+                switch (this.viewModel.settings.dateType.type) {
+                    case "Second":
```
Comment (uve) [inline src/gantt.ts]: Please move all hardcoded string to ENUMs

Comment (savvinsergey) [reply src/gantt.ts]: DONE

File: src/gantt.ts [hunk 2]
```diff
@@ -1407,10 +1418,27 @@ module powerbi.extensibility.visual {
...
+            timestamp: number,
+            milestoneTitle: string): VisualTooltipDataItem[] {
+            let dateTime: string = new Date(timestamp).toLocaleDateString();
+
+            if (!milestoneTitle) {
+                switch (this.viewModel.settings.dateType.type) {
+                    case DateTypes.Second:
+                    case DateTypes.Minute:
+                    case DateTypes.Hour:
+                        milestoneTitle = "Now";
```
Comment (uve) [inline src/gantt.ts]: Do the same for strings "Now" and "Today"

Comment (savvinsergey) [reply src/gantt.ts]: DONE

**Pull request merged**